### PR TITLE
Trial

### PR DESCRIPTION
**Pull request by HariniRamachandren**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "**/.*/**",
+    "**/.vscode",
+    "**/.vscode/**"
+  ],
+  "C_Cpp_Runner.useAddressSanitizer": false,
+  "C_Cpp_Runner.useUndefinedSanitizer": false,
+  "C_Cpp_Runner.useLeakSanitizer": false,
+  "C_Cpp_Runner.showCompilationTime": false,
+  "C_Cpp_Runner.useLinkTimeOptimization": false
+}
```
Comment (vinodchinthoti) [inline .vscode/settings.json:58]: okay, it seems generated from VS.

File: .vscode/c_cpp_properties.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "windows-gcc-x86",
+      "includePath": [
+        "${workspaceFolder}/**"
+      ],
+      "compilerPath": "C:/MinGW/bin/gcc.exe",
+      "cStandard": "${default}",
+      "cppStandard": "${default}",
+      "intelliSenseMode": "windows-gcc-x86",
+      "compilerArgs": [
+        ""
```
Comment (vinodchinthoti) [inline .vscode/c_cpp_properties.json:13]: is it json file?

**Review by vinodchinthoti — COMMENTED** [review]
see the comments

File: .vscode/c_cpp_properties.json (hunk 2 above)
Comment (HariniRamachandren) [reply .vscode/c_cpp_properties.json:13]: s sir

**Review by vinodchinthoti — CHANGES_REQUESTED** [review]
check the other comment.

**Review by vinodchinthoti — CHANGES_REQUESTED** [review]
modify code

**Review by vinodchinthoti — APPROVED** [review]
merge

**Pull request merged**